meerk40t/gui/scene/scenewidgets.py

```diff
@@ -618,7 +618,7 @@ def event(self, window_pos=None, space_pos=None, event_type=None):
                 sy = self.start_location[1]
                 ex = self.end_location[0]
                 ey = self.end_location[1]
-                right_drag = sx <= ex and sy <= ey
+                right_drag = sx <= ex # and sy <= ey
```

Comment:
Without sy <= ey, isn't this purely right drag. So you right click draw a purely horizontal line to the left, and everything in the scene within that line horizontally would be selected? This seems weird. I was told other graphics utilities used this sort of right-up drag function to select objects and left-down as a functional and useful difference but I've not really heard of it without also checking y. That seems like it would be weird.